README.md

```diff
@@ -0,0 +1,15 @@
+# wasp-video-stream-service
+
+## Getting Started
+
+### Requirements
+
+### Building
+
+### Testing
+
+### Linting
+
```

Comment:
Is there any commands you can put under these headings now they're in the makefile?

Reply:
I've added a few but things like make build just produce a binary which Dave has already told me off about!

Reply:
Added some commands